kmir/src/kmir/kdist/mir-semantics/symbolic/kmir-symbolic-locals.md

```diff
@@ -0,0 +1,98 @@
+# Symbolic locals for functions
+
+This module contains rules that follow the same structure as `#init` in the KMIR module, however it reads the arguments for the signature of the start symbol and generates symbolic values for them.
+
+```k
+requires "../kmir.md"
+
+module KMIR-SYMBOLIC-LOCALS [symbolic]
```

Comment:
When reading this, I liked the idea of moving all initialisation functions from `KMIR-CONTROL-FLOW` into a new module that control flow can call into.
However, we might have to keep every symbolic rule (that seems to mean involving existentials here) in its own symbolic module? 
I don't see why `#initSymbolic` needs to be different from `#init` otherwise, we simply should not execute a function that has arguments without supplying any.